### PR TITLE
2 traductions

### PR DESCRIPTION
**Pull request by AxioBen**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -176,6 +176,9 @@
     <script src="bower_components/pdfmake/build/pdfmake.js"></script>
     <script src="bower_components/pdfmake/build/vfs_fonts.js"></script>
     <script src="bower_components/angular-file-upload/dist/angular-file-upload.min.js"></script>
+    <script src="bower_components/angular-dynamic-locale/tmhDynamicLocale.min.js"></script>
```
Comment (fabian) [inline app/index.html]: @AxioBen I see you added a new dependency `angular-dynamic-locale` - but I can't see that dependency in `bower.json`, can you add it there too?

Comment (AxioBen) [reply app/index.html]: @fabian Done in aa922d62667066d7b78c90968d1c7fbb15cbaed2

**Comment by WorldSkillsFrance** [comment]
Can we validate this issue @fabian please

**Comment by fabian** [comment]
@WorldSkillsFrance I've merged the changes now and will deploy them on Monday to our production environment.

**Pull request merged**